### ISSUE TITLE
Display a system notification when video is ready to publish

### ISSUE THREAD

**Issue by Booteille** [Chocobozzz/PeerTube#703]
I think that would be useful if we could get notified when the video finished to upload and is ready to be published.

What do you think about?

**Comment by rigelk** [comment]
Parent issue is #493, most likely blocking.

But it's not like the client interface doesn't show you a progress bar. Maybe you refer more to desktop notification to have the possiblity to leave the tab open and forget about it while a long video uploads, and then have a system notification?

**Comment by Booteille** [comment]
Yup, I was thinking about desktop notification but couldn't remember the right name. Ah ah.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/Chocobozzz/PeerTube/issues/703#issue-335006950

@Booteille, I'd prefer to have this sent via e-mail.
